client/app/controllers/auth.js

```diff
@@ -42,4 +43,8 @@ angular.module('app.auth', ['app.services', 'app.eventfactory'])
       });
   };
 
+  function checkForm(data) {
```

Comment:
No. That's a stub. I'm buiding out a function to check text fields against
regular expressions to filter out bad inputs. I think that's what's causing
severe lag in signing up

On Wed, Jul 27, 2016, 2:03 PM Nate Blain <notifications@github.com> wrote:

> In client/app/controllers/auth.js
> <https://github.com/DNAExpress/dnaexpress/pull/207#discussion_r72523264>:
>
> > @@ -42,4 +43,8 @@ angular.module('app.auth', ['app.services', 'app.eventfactory'])
> >        });
> >    };
> >
> > +  function checkForm(data) {
>
> What is this function for? Is it necessary to keep this log in here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DNAExpress/dnaexpress/pull/207/files/ac85f6e27352e389f48a952fbbbeef24d20eb651#r72523264>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOeOveZ9jSv_429b3y7wkQLsov_VuF06ks5qZ8eugaJpZM4JWnHR>
> .
>